doc/AnalysisWithJQ-Japanese.md

```diff
@@ -574,4 +575,25 @@ Sysmonは通常、`MD5`、`SHA1`、`IMPHASH`など複数のハッシュを計算
 
 たとえば、次のようにMD5ハッシュを抽出して重複を削除できます:
 
-`cat results.json | jq 'select ( .Details.Hashes? ) | .Details.Hashes | .[4:36] ' -r | sort | uniq`
\ No newline at end of file
+`cat results.json | jq 'select ( .Details.Hashes? ) | .Details.Hashes | .[4:36] ' -r | sort | uniq`
+
+## 9. PowerShellログの抽出
+
+PowerShellのScriptblockログ（EID: 4104）は、通常多くのログに分割されており、CSV形式で出力する際、Hayabusaはタブやリターン文字を削除して簡潔に出力します。
+しかし、PowerShellログは、本来のタブとリターン文字の書式で、ログを組み合わせて解析するのが最も簡単です。
+ここでは、VSCode等で開いて分析するために、`COMPUTER-A`からPowerShell EID 4104のログを抽出して、`.txt`ファイルに保存する例を紹介します。
+ScriptBlockフィールドを抽出した後、`awk`を使って`\r\n`と`\n`をリターン文字に、`\t`をタブに置き換えています。
+
+```
+cat results.json | jq 'select ( .EventID == 4104 and .Details.ScriptBlock? != "n/a" ) | .Details.ScriptBlock , "\r\n"' -j | awk '{ gsub(/\\r\\n/,"\r\n"); print; }' | awk '{ gsub(/\\t/, "\t"); print; }' | awk '{ gsub(/\\n/, "\r\n"); print; }' > 4104-PowerShell-Logs.txt
```

Comment:
Thank you for nice example :)
This command seems to require the following filter for `COMPUTER-A`, what do you think? 
`select ( .EventID == 4104 and .Details.ScriptBlock? != "n/a" and .Computer == "COMPUTER-A.hinokabe.local"`

Reply:
thank you! nice catch. I forgot I deleted that when testing. I put it back.